cmd/gossamer/commands/root.go

```diff
@@ -162,8 +168,17 @@ func addRootFlags(cmd *cobra.Command) error {
 	}
 
 	// Log Config
-	if err := addLogFlags(cmd); err != nil {
-		return fmt.Errorf("failed to add log flags: %s", err)
+	if err := addStringFlagBindViper(
+		cmd, "logs", "",
```

Comment:
- Let's use `--log` and the shorthand `-l` since that's how it is defined in substrate
- Let's specify the available modules in the usage
- We don't need to bind it to viper since we are parsing it. Let's not use `addStringFlagBindViper`. Here's an [example](https://github.com/ChainSafe/gossamer/blob/development/cmd/gossamer/commands/root.go#L251)